ServerCore/Pages/Shared/_Layout.cshtml

```diff
@@ -22,19 +17,17 @@
     </environment>
 
     <!-- TODO: Make this actually scalable and correct -->
-    @if (ev != null)
-    {
-        <link rel="stylesheet" href="~/css/@(ev.UrlString)/styles.css" />
+    @{
+        // Using ev here so that it isn't the same as Event that we bind in the <body>
+        // Works fine if we use Event, but VS will yell at you with an error in the IDE
+        var ev = (Model as ServerCore.ModelBases.EventSpecificPageModel)?.Event;
+        if (ev != null && ev.UrlString == "pd2019")

```

Comment:
Could you probe for the file or something instead of special-casing this event so this works equally well next year?

Reply:
Tried to use a File.Exists but that required a fully-qualified path, or at least that's what I thought. Will keep digging.